ecl/hql/hqlgram2.cpp

```diff
@@ -12003,3 +12362,186 @@ IHqlExpression *HqlGram::doParse()
     return NULL;
 }
 
+/*The following confusion  matrix holds the cost of transpositions and substitutions for weighted-Damerau-Levenshtein Distance.
```

Comment:
This code actually belongs to https://github.com/hpcc-systems/HPCC-Platform/pull/8130 but wasn't sure on how to incorporate since the code in that PR is within the plugins?